Gems/AtomLyIntegration/CommonFeatures/Code/Source/Grid/GridComponentController.h

```diff
@@ -80,4 +80,4 @@ namespace AZ
             bool m_dirty = true; // must be set to true for any configuration change that rebuilds the grid
         };
     } // namespace Render
-} // namespace AZeneccccckfdljrndicnfefjdklthhfifnnvkffebgkbd
```

Comment:
What happened here? :P

Reply:
It was in there since the beginning of the repo